helper/keysutil/policy.go

```diff
@@ -326,6 +326,13 @@ type Policy struct {
 	// a max.
 	ArchiveVersion int `json:"archive_version"`
 
+	// ArchiveMinVersion is the minimum version of the key in the archive.
+	ArchiveMinVersion int `json:"archive_min_version"`
```

Comment:
This seems redundant. Archiving is simply a factor of the min decryption version. So if the min decryption version is the same as the min version (or, first available version), nothing is in the archive; otherwise, the first version in the archive is the same as min version.

Reply:
I don't feel that this is redundant (unless I am failing to see your point). The policy that invokes `handleArchiving` might already have `MinAvailableVersion` modified. So, we can't rely on that. We want the _last used_ MinAvailableVersion, which is held by ArchiveMinVersion.

Reply:
Gotcha, sounds good then.